api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -163,13 +163,13 @@ paths:
             "directing department".
           schema:
             type: string
-            examples:
-              ASC:
-                value: 'ASC'
-                summary: Will select all (parent) Signals where ASC is the directing department
-              null:
-                value: 'null'
-                summary: Will select all (parent) Signals withouta directing department assignemt
+          examples:
+            ASC:
+              value: 'ASC'
+              summary: Will select all (parent) Signals where ASC is the directing department
```

Comment:
consider adding: "set explicitly" (as opposed to implicitly because through responsible department)

Reply:
Added